src/utils/utils.ts

```diff
@@ -15,6 +15,7 @@ import {
 import chalk from 'chalk'
 import Token from '@nevermined-io/nevermined-sdk-js/dist/node/keeper/contracts/Token'
 import ERC721 from '../abis/ERC721.json'
+import NFTUpgradeable from '@nevermined-io/contracts/artifacts/NFTUpgradeable.spree.json'
```

Comment:
This should be added the same way as ERC721 abi. It would be best if they both were in sdk-js.